### ISSUE TITLE
Add support for expression objects

### ISSUE THREAD

**Issue by cdavernas** [serverlessworkflow/specification#899]
**What would you like to be added**:

Provide support for using both object and string expressions for `input.from`, `export.as` and `output.as`.

**Why is this needed**:

- Convenience
- Improved readability for complex expressions

*Currently:*
```yaml
...
output: 
  as: '{ style: { shape: "circle", backgroundColor: .colors[0] } }'
```

Proposal:
```yaml
...
output:
  as:
    shape: circle
    style:
      backgroundColor: ${ .colors[0] }
```

**Comment by ricardozanini** [comment]
+1000 are you working on it?

**Comment by cdavernas** [comment]
@ricardozanini I'll open a PR. It's actually already supported by https://github.com/serverlessworkflow/synapse/tree/1.0.0-alpha1.

**Comment by ricardozanini** [comment]
Just adding the assignee then to facilitate planning :P

**Comment by cdavernas** [comment]
@fjtirado @JBBianchi @matthias-pichler-warrify sounds good to you guys?

**Comment by JBBianchi** [comment]
Good call!

**Comment by fjtirado** [comment]
Sure, makes sense.

**Comment by matthias-pichler-warrify** [comment]
looks good 👍

**Comment by cdavernas** [comment]
Closed by #907
